src/main/java/org/geysermc/discordbot/commands/ExploitReportCommand.java

```diff
@@ -0,0 +1,77 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/GeyserDiscordBot
+ */
+
+package org.geysermc.discordbot.commands;
+
+import com.google.common.collect.ImmutableMap;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jdautilities.command.SlashCommand;
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.components.actionrow.ActionRow;
+import net.dv8tion.jda.api.components.buttons.Button;
+import net.dv8tion.jda.api.components.buttons.ButtonStyle;
+import net.dv8tion.jda.api.components.container.Container;
+import net.dv8tion.jda.api.components.section.Section;
+import net.dv8tion.jda.api.components.separator.Separator;
+import net.dv8tion.jda.api.components.textdisplay.TextDisplay;
+import net.dv8tion.jda.api.components.thumbnail.Thumbnail;
+import net.dv8tion.jda.api.interactions.InteractionContextType;
+import net.dv8tion.jda.api.interactions.commands.Command;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.geysermc.discordbot.util.BotColors;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class ExploitReportCommand extends SlashCommand {
+
+    public ExploitReportCommand() {
+        this.name = "exploitreport";
+        this.help = "Sends the message to allow users to report exploits";
+        this.contexts = new InteractionContextType[]{InteractionContextType.GUILD};
+        this.userPermissions = new Permission[]{Permission.MANAGE_SERVER};
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        event.getChannel().sendMessageComponents(Container.of(
+                        TextDisplay.of("## Report an exploit"),
+                        TextDisplay.of("Found an exploit in Geyser? Please report it to us!\n\nPress the button below to open a new private report."),
```

Comment:
```suggestion
                        TextDisplay.of("Found an exploit in Geyser? Please report it to us!\n\nPress the button below to open a new private report.\n\n
This is NOT a support thread! Misuse will be punished."),
```
maybe something like that to try and limit the amount of users misusing it as help